tests/Propel/Tests/Generator/Behavior/Sluggable/SluggableBehaviorTest.php

```diff
@@ -111,6 +111,8 @@ public static function cleanupSlugProvider()
     public function testObjectCleanupSlugPart($in, $out)
     {
         $t = new TestableTable13();
+        // locale must be utf8 for be sure test work (iconv)
+        setlocale(LC_CTYPE, 'en_GB.utf8');
```

Comment:
this line changes global state... we either need to reset the state after this test or we need to define this locale in a more global place so it works for all tests (and we dont have global state sideeffects after this particular test ran)